x-pack/plugins/ml/public/jobs/jobs_list/components/job_details/forecasts_table/_forcasts_table.scss

```diff
@@ -2,31 +2,20 @@
 .forecasts-table {
```

Comment:
Just a reminder. This doesn't look needed. Might want to just look into using the `compressed` prop on the table.

Reply:
Good spot @snide. I have replaced the two forecast table scss files with JS props on the table. Can you take another look please.